src/bridge/cockpitcpusamples.h

```diff
@@ -26,6 +26,11 @@ G_BEGIN_DECLS
 
 void            cockpit_cpu_samples         (CockpitSamples *samples);
 
+gchar*          read_hwmon_file             (const gchar *path);
+
+void            sample_cpu_sensors          (CockpitSamples *samples, int hwmonID);
```

Comment:
Please drop these -- this isn't API for other parts of cockpit. Make the functions static instead (see above).

Reply:
Done.